Functions.Templates/Documentation/twilioSmsOut.md

```diff
@@ -1 +1,163 @@
-This feature is experimental. Docs will be coming soon!
\ No newline at end of file
+## Settings for a Twilio output binding
+
+- `type` : must be set to *twilioSms*.
+- `direction` : must be set to *out*.
+- `name` : Variable name used in function code for the Twilio SMS text message. 
+- `accountSidSetting` : This value must be set to the name of an app setting that holds your Twilio Account Sid e.g. TwilioAccountSid. If not set, the default app setting name is "AzureWebJobsTwilioAccountSid". 
+- `authTokenSetting` : This value must be set to the name of an app setting that holds your Twilio authentication token e.g. TwilioAccountAuthToken. If not set, the default app setting name is "AzureWebJobsTwilioAuthToken". 
+- `from` : This value is set to the phone number that the SMS text is sent from
+- `body` : This value can be used to hard code the SMS text message if you don't need to set it dynamically in the code for your function. 
+
+## Example - Functions 2.x
+
+See the language-specific example:
+
+* [2.x C# script (.csx)](#2x-c-script-example)
+* [2.x JavaScript](#2x-javascript-example)
+
```

Comment:
Nit: Don't need "2.x" explicitly
You can do * [C# script (.csx)](#c-script-example)